src/displayBackend/wayland/WaylandZCopy.cpp

```diff
@@ -379,10 +379,16 @@ void WaylandLinuxDmabuf::sOnModifiers(void *data,
 	 * We do not support passing modifiers from the frontend to the backend,
 	 * so ignore all formats with modifiers set.
 	 */
-	if (modifierHi == 0 && modifierLo == 0)
+	/*
+	 * If WITH_MODIFIERS_HACK is defined, we will skip modifiers check.
```

Comment:
We need to elaborate this comment: at least describe why it someone would need this enabled.